gatsby-config.js

```diff
@@ -8,10 +8,10 @@
 
 module.exports = {
   siteMetadata: {
-    title: 'React: Eine JavaScript Bibliothek für das Bauen von Benutzeroberflächen',
-    siteUrl: 'https://reactjs.org',
+    title: 'React: Eine JavaScript Bibliothek für das Erstellen von Benutzeroberflächen',
+    siteUrl: 'https://de.reactjs.org',
     rssFeedTitle: 'React',
-    rssFeedDescription: 'Eine JavaScript Bibliothek für das Bauen von Benutzeroberflächen',
+    rssFeedDescription: 'Eine JavaScript Bibliothek für das Erstellen von Benutzeroberflächen',
```

Comment:
sounds way better! :fireworks: 

optional addition:
```suggestion
    rssFeedDescription: 'Eine JavaScript Bibliothek zum Erstellen von Benutzeroberflächen',
```

Reply:
right! sounds more colloquial.